assets/character/scripts/character_action_plant_crop.gd

```diff
@@ -26,7 +24,6 @@ func abort():
 		nav_to_action.abort()
 
 func plant_crop():
-	var crop: Crop = crop_scn.instantiate()
-	character.add_sibling(crop)
-	GameManager.change_item_count(plant_seed, -1)
-	crop.initialise(plant_cell, false, plant_seed)
+	GameManager.plant_crop(plant_seed, plant_cell)
+	if character is PlayerCharacter:
+		GameManager.change_item_count(plant_seed, -1)
```

Comment:
Good catch

Reply:
Yep, later on if/when all characters have their own 'items held' we need a check to remove it from them too